generation/WinSDK/autoTypes.json

```diff
@@ -80,8 +80,9 @@
     "NativeTypedef": true
   },
   {
-    "Namespace": "Windows.Win32.Foundation",
+    "Namespace": "Windows.Win32.Security",
```

Comment:
PSID was probably in Foundation to avoid circular namespace dependencies. Did we regress anything here?

Reply:
Unknown. Tests are green and I ran this through Rust bindings with no ill effects. (We'll need to update WDK metadata.) I don't think metadata guarantees non-circular namespace dependencies, does it?

Reply:
@kennykerr are you concerned about this change at all? We have a test for cyclical namespaces, so if that passed then it might be ok, but would be good to sanity check on your end.

Reply:
Yep all good on my end.

Reply:
An alternative, if this becomes an issue, is to move the functions into Foundation. But that seems wrong given there's nothing foundational about SIDs.

Reply:
Right, it's a security API. The cycle thing never amounted to much. I wouldn't worry about it.